.github/workflows/docker-builds.yml

```diff
@@ -31,31 +31,36 @@ permissions: read-all
 
 jobs:
   docker-build:
-    runs-on: [self-hosted, linux.12xlarge]
     environment: ${{ (github.ref == 'refs/heads/main' || startsWith(github.event.ref, 'refs/tags/v')) && 'docker-build' || '' }}
     timeout-minutes: 240
     strategy:
       fail-fast: false
       matrix:
+        runner: [linux.12xlarge]
+        docker-image-name: [
+          pytorch-linux-focal-cuda12.1-cudnn8-py3-gcc9,
+          pytorch-linux-focal-cuda12.1-cudnn8-py3-gcc9-inductor-benchmarks,
+          pytorch-linux-focal-cuda11.8-cudnn8-py3-gcc9,
+          pytorch-linux-focal-py3.8-clang10,
+          pytorch-linux-focal-py3.11-clang10,
+          pytorch-linux-focal-py3.12-clang10,
+          pytorch-linux-focal-rocm-n-1-py3,
```

Comment:
Is it still possible to stick to the block sequence stile, i.e.
`docker-image-name:
      - a 
      - b
      - c
```